ycsb/main.go

```diff
@@ -466,9 +448,12 @@ func main() {
 
 	loadStart := time.Now()
 	var wg sync.WaitGroup
-	for i := range workers {
+	// TODO(peter): Using all of the workers for loading leads to errors with
+	// some of the insert statements receiving an EOF. For now, use a single
+	// worker.
+	for i, n := 0, 1; i < n; i++ {
```

Comment:
It's fine if we actually plan on coming back and changing it, but this is a really strange way of doing something once.